components/message/nz-message-config.ts

```diff
@@ -7,6 +7,9 @@ export interface NzMessageConfig {
   nzAnimate?: boolean;
   // For message container only
   nzMaxStack?: number;
+  nzTop?: string;
```

Comment:
nzMessage does not support nzTop | nzBottom | nzPlacement? any reason add these properties here?

Reply:
It was because of [`nz-notification-container.component.html`](https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/components/notification/nz-notification-container.component.html) using those properties, although I just realised that component is working with `NzNotificationConfig`, which extends `NzMessageConfig`.

I've updated the PR to reflect that in [`NzNotificationContainerComponent`](https://github.com/NG-ZORRO/ng-zorro-antd/pull/2554/files#diff-643edfe948469ef057fd967d81646a56R15), and I've also rebased it on `master`.